### DISCUSSION TITLE
Is it possible to save dashboard to Favorites (Starred) together with current variable values?

### DISCUSSION THREAD

**Discussion by EHabarov** [grafana/grafana#69708] [Dashboards, Panels & Visualizations]
Hi,

We have a dashboard  for a group of users which shows a list of processes.
Most users have only "Viewer" permissions.
Dashboard contents can be filtered with variable to show only subset of processes.
Each user uses it's own value for filter variable to watch only required processes.
But, each time when dashboard is opened from dashboard list or favorites list - common default value of variable is applied. 

Is it possible to save dashboard to favorites/starred for current user inside Grafana together with variable values? 
Or first "star" dashboard and then edit a link to it to include variable values?
Maybe it can or should be done some other way inside Grafana?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
